bindings/nodejs/generated.js

```diff
@@ -36,7 +36,7 @@ function isMusl() {
   // For Node 10
   if (!process.report || typeof process.report.getReport !== 'function') {
     try {
-      const lddPath = require('child_process').execSync('which ldd').toString().trim();
+      const lddPath = require('child_process').execSync('which ldd').toString().trim()
```

Comment:
This file should not be touched.

Reply:
I didn't intent to modify this file. But everytime i run `yarn build`, This file's license header will be remove and the content will be changed. So i have manually add it back.  😢

Reply:
Oh, it's our fault. We will fix it later.

Reply:
I'll handle this.